shared/src/userData.ts

```diff
@@ -29,6 +29,7 @@ export const createEmptyUserData = (user: BusinessUser): UserData => {
       roadmapOpenSteps: [],
       hiddenCertificationIds: [],
       hiddenFundingIds: [],
+      hiddenRoadmapSidebarCards: ["successful-registration", "not-registered"],
```

Comment:
Why do these start hidden? Shouldn't this be controlled by whether they are registered or not?